contracts/HtsSystemContract.sol

```diff
@@ -845,22 +843,23 @@ contract HtsSystemContract is IHederaTokenService {
     function _balanceOfSlot(address account) internal virtual returns (bytes32) {
         bytes4 selector = IERC20.balanceOf.selector;
         uint192 pad = 0x0;
-        uint32 accountId = HtsSystemContract(HTS_ADDRESS).getAccountId(account);
+        (uint32 accountId, ) = IHederaAccounts(HTS_ADDRESS).getAccountId(account);
```

Comment:
If we use `getAccountId` directly from `HtsSystemContract`, we don't need `IHederaAccounts`.